src/index.js

```diff
@@ -13,6 +13,25 @@ import requireReturnsDescription from './rules/requireReturnsDescription';
 import requireReturnsType from './rules/requireReturnsType';
 
 export default {
+  configs: {
+    recommended: {
+      rules: {
+        'jsdoc/check-param-names': 'warn',
+        'jsdoc/check-tag-names': 'warn',
+        'jsdoc/check-types': 'warn',
+        'jsdoc/newline-after-description': 'warn',
+        'jsdoc/require-description-complete-sentence': 'off',
+        'jsdoc/require-example': 'off',
+        'jsdoc/require-hyphen-before-param-description': 'off',
+        'jsdoc/require-param': 'warn',
+        'jsdoc/require-param-description': 'warn',
+        'jsdoc/require-param-name': 'warn',
+        'jsdoc/require-param-type': 'warn',
+        'jsdoc/require-returns-description': 'warn',
+        'jsdoc/require-returns-type': 'warn'
```

Comment:
ESLint also has 2 default JSDoc-esque rules: [**require-jsdoc**](https://eslint.org/docs/rules/require-jsdoc) and [**valid-jsdoc**](https://eslint.org/docs/rules/valid-jsdoc). Wasn't sure if we should enable those in the recommended rule set or not, so I opted for not -- since I wasn't sure how/if the ESLint rules would conflict with this module.